### ISSUE TITLE
Include parameters in output

### ISSUE THREAD

**Issue by joshhunt** [sharkdp/hyperfine#704]
I'm trying to benchmark the same command's performance in two difference git branches, using parameters to switch between the branches:

```bash
hyperfine --warmup 1 --runs 5 -L branch main,upgrade --prepare "git co {branch}; yarn" "yarn tsc --noEmit"
```

However, at the end the output doesn't make it clear which parameter performed better

```
Benchmark 1: yarn tsc --noEmit
  Time (mean ± σ):     12.865 s ±  0.067 s    [User: 18.382 s, System: 1.247 s]
  Range (min … max):   12.785 s … 12.966 s    5 runs

Benchmark 2: yarn tsc --noEmit
  Time (mean ± σ):     12.033 s ±  0.092 s    [User: 16.474 s, System: 1.463 s]
  Range (min … max):   11.965 s … 12.189 s    5 runs

Summary
  'yarn tsc --noEmit' ran
    1.07 ± 0.01 times faster than 'yarn tsc --noEmit'
```

It would be great if the output included the value of the parameter.

As a workaround, I can make the benchmark command something like `"yarn tsc --noEmit; echo '{branch}'"` just to distinguish between the two

**Comment by sharkdp** [comment]
I think this should be supported since https://github.com/sharkdp/hyperfine/pull/644, which landed in hyperfine 1.17

**Comment by joshhunt** [comment]
Ahhh, I should have checked! 

I did try searching for issues before filing this, but didn't find https://github.com/sharkdp/hyperfine/issues/600 for some reason.

Thanks!
